components/Flipper/FlipRestrictionList/FlipRestrictionList.module.css

```diff
@@ -36,6 +36,21 @@
     font-weight: bold;
 }
 
+.sortByNameCheckbox{
+    width: 49%;
+    position: relative !important;
+}
+
+.sortByNameContainer{
+    display: inline-flex;
+    margin-left: 20px;
+}
+
+.searchFilter{
+    width: 49%;
```

Comment:
why is this 49%?
I assume you want to make it half of the container?
How about flexbox/css-gird?